ui/src/components/pages/Namespace/index.tsx

```diff
@@ -1,219 +1,111 @@
-import { useEffect, useState, useContext } from "react";
-import { AppContext, AppContextProps } from "../../../App";
-import { NamespaceRowContent } from "./partials/NamespaceRowContent";
-import {
-  TableBody,
-  Table,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TableRow,
-  Paper,
-  TextField,
-  Autocomplete,
-  InputAdornment,
-  Popper,
-} from "@mui/material";
+import React, { useEffect, useState, useMemo } from "react";
+import { useParams } from "react-router-dom";
 import Box from "@mui/material/Box";
-import List from "@mui/material/List";
-import ListItem from "@mui/material/ListItem";
-import SearchIcon from "@mui/icons-material/Search";
-import { notifyError } from "../../../utils/error";
-import { useNamespaceListFetch } from "../../../utils/fetchWrappers/namespaceListFetch";
-import { SetNamespaceList, SetStore } from "../../../localStore/SetStore";
-import { GetStore } from "../../../localStore/GetStore";
+import CircularProgress from "@mui/material/CircularProgress";
+import { useNamespaceSummaryFetch } from "../../../utils/fetchWrappers/namespaceSummaryFetch";
+import { SummaryPageLayout, SummarySection, SummarySectionType } from "../../common/SummaryPageLayout";
 
 import "./style.css";
 
-const NS_Curr_Store = "curr_namespace";
-const NS_List_Store = "namespace_list";
+const DATA_REFRESH_INTERVAL = 15000; // ms
```

Comment:
for other APIs for graph we refresh at 30sec and 1min, should we set a standard refresh interval for all?

Reply:
Let's not have a single refresh interval for all. The pipeline view will be the most aggressive one, while others can be a little slower. Refresh at a cluster level is the most resource-intensive one with the least ROI and hence can be slower.